### ISSUE TITLE
Extract Gecko repository informations from about:buildconfig's Source section

### ISSUE THREAD

**Issue by xabolcs** [mozilla/nightlytt#129]
In Thunderbird and SeaMonkey the SourceStamp from `application.ini`
is different from changeset which on `about:buildconfig` page can be found.

For example, see @tonymec's buildid template in his [comment](https://github.com/mozilla/nightlytt/issues/125#issuecomment-15859094) below!
He uses both changeset as " ..... c-c:0123456789ab m-c:456789abcdef"

> Works for me. I'm using the latest bleeding-edge NTT ("master" of the mozilla/nightlytt repo, currently 3.5pre-201303200004) and I use a slightly modified nightly.templates.buildid, with the value "${UserAgent} ID:${AppBuildID}${Flags} c-c:${Changeset}" then I usually paste the mozilla-central changeset from about:buildconfig, as follows (where ${Flags} is empty and m-c and what follows are added manually):
> 
> Mozilla/5.0 (X11; Linux x86_64; rv:23.0) Gecko/20100101 Firefox/23.0 SeaMonkey/2.20a1 ID:20130403003001 c-c:4ad03358510a m-c:97cfc16ba5dc
> 
> The timestamp in the .xpi name is CET (GMT+0100), since the switch to summer time happened here in the night from 30 to 31 March this year.

This issue is all about this:
make @tonymec's (and others) life easier providing the `GeckoChangeset` too. :)

**Comment by whimboo** [comment]
With the rewrite of the extension we have to rely on what the WebExtension API is handing us over. So if something is not working for that new release, please file a new issue for it.

**Comment by xabolcs** [comment]
I don't understand the reasoning about closing this issue.

It's is blocked by #246 instead of closing. Hm?

**Comment by whimboo** [comment]
Do we know what the WebExtension API returns in case of SeaMonkey as build information? It may contain the correct data. The originally filed approach to read it from `application.ini` file will not work anytime longer.
